app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -47,6 +48,11 @@ public void onLoginButtonClicked(String email, String pass, Context context) {
         mPassword = pass;
         mContext = context;
```

Comment:
I should comment this in a previous PR, but if it will be used only in `isNetworkConnected`, let send it directly to the method by parameter. If you set it as an attribute of the `LoginPresenter` it will stay alive and consuming memory until the presenter is killed. The same happen, I think, with `mPassword`, `mUser` and `mUsers`.